server/webapp/WEB-INF/rails.new/app/helpers/pipelines_helper.rb

```diff
@@ -30,7 +30,7 @@ def run_stage_label stage
 
   def trigger_message(actual_date_time, pim)
     if pim.class != com.thoughtworks.go.presentation.pipelinehistory.PreparingToScheduleInstance
-      "Triggered&nbsp;#{l.string('BY')}&nbsp;<span class='who'>#{h pim.getApprovedBy()}</span>&nbsp;on&nbsp;<span class='time'>#{actual_date_time.to_long_display_date_time()}</span><input type='hidden' value='#{actual_date_time}'/>"
+      "Triggered&nbsp;#{l.string('BY')}&nbsp;<span class='who'>#{h pim.getApprovedBy()}</span>&nbsp;on&nbsp;<span class='time'></span><input type='hidden' value='#{actual_date_time}'/>"
```

Comment:
I am not sure why was it there in the first place, so just left it. I can check.